regxmllib/CMakeLists.txt

```diff
@@ -44,3 +45,5 @@ endforeach( UNIT_TEST_PATH ${UNIT_TESTS} )
 #add_executable(AUIDTest src/test/cpp/com/sandflow/smpte/util/AUIDTest.cpp)
 #target_link_libraries(AUIDTest regxmllibc)
 #add_test(AUIDTest AUIDTest)
+
+install(DIRECTORY ${CMAKE_SOURCE_DIR}/src/main/cpp/com/sandflow DESTINATION include FILES_MATCHING PATTERN "*.h")
```

Comment:
`install(DIRECTORY src/main/cpp/com/sandflow DESTINATION include FILES_MATCHING PATTERN "*.h")`

should work as well, right? (without `${CMAKE_SOURCE_DIR}/`)

Reply:
yes that would work as well. I just included ${CMAKE_SOURCE_DIR}/ for readability but we can remove that. Let me update the pull request.